.github/workflows/issue-fields-backfill.yml

```
@@ -74,13 +75,14 @@ jobs:
 
       - name: Create GitHub App token
         id: app-token
-        uses: actions/create-github-app-token@v2
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)